### PR TITLE
feat: prism is more than just mocking

### PR DESCRIPTION
**Pull request by philsturgeon**
Adding callbacks and validation proxy to the site, but still need some images. Prism is a very CLIish tool and i dunno if we can show much off here. 

@paulatulis @XVincentX Do we go with random stock images? 

image

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 59cf83b5ec1541569f768995f2a901190bd21086

https://deploy-preview-1331--stoplightio.netlify.com

**Comment by ksatx** [comment]
@philsturgeon these are some of the images I try when I need a visual for Prism but don't have one.

colored-flasks-med
image

**Comment by philsturgeon** [comment]
Ok using that for validation proxy but what about callbacks? Is it this? I want it to be this. 
image

**Comment by ksatx** [comment]
I think any telephone works ¯\_(ツ)_/¯

btw, I've got Canva Photos Unlimited, so hit me up whenever we need a stock photo and don't want to be obligated to include an unsplash credit

OVERSEASCALL

**Comment by philsturgeon** [comment]
Probably this one? 
https://www.canva.com/photos/MAC8IJB1UL8-call-me-note/

File: netlify/landings/prism.yaml [hunk 1]
```diff
@@ -58,20 +58,32 @@ featureSection:
...
-    - description: >-
-        In case you're wondering why Prism has responded to a request in such way,
-        the logs show you the entire negotiation process, so you can modify the spec
-        or your code quickly.
+    - description: >
+        Requests and responses take care of a lot of API interactions, but what 
+        about callbacks? OpenAPI v3.0 allows API designers to define callbacks, 
+        and construct a URL to fire a payload at, so Prism will too. Notice HTTP
+        callbacks coming in, allowing you to attempt to integrate them before the
+        API has even been built.
```
Comment (paulatulis) [inline netlify/landings/prism.yaml]:
```suggestion
        Requests and responses take care of a lot of API interactions, but what about callbacks? With OpenAPI v3.0, API designers can define callbacks and construct a URL at which to fire a payload. And so can Prism. With the ability to notice incoming HTTP callbacks, you're now free to integrate with those callbacks before an API has even been built.
```

File: netlify/landings/prism.yaml [hunk 2]
```diff
@@ -58,20 +58,32 @@ featureSection:
...
       image: /images/prism_logging.png
-      shortName: Logging
-      title: Extensive logging
+      shortName: Callbacks
+      title: Mocking Callbacks
+
+    - description: >
+        Identify discrepancies between the OpenAPI document and target API to
+        help frontend developers integrating with your API. The proxy can also
```
Comment (paulatulis) [inline netlify/landings/prism.yaml]:
```suggestion
        help frontend developers integrate with your API. The proxy can also
```

File: netlify/landings/prism.yaml [hunk 3]
```diff
@@ -24,14 +24,14 @@ hero:
...
     enabled: true
     text: Do you want to mock your API in minutes?
 
   title: Work with your API before you write any code
   description: >-
-    Prism is an open source HTTP mock server that can mimic your API's behavior as if you already built it. Mock HTTP servers are generated from your OpenAPI v2/v3 (formerly known as Swagger) documents.
+    Prism is an open-source HTTP mock server that can mimic your API's behavior as if you already built it. Mock HTTP servers are generated from your OpenAPI v2/v3 (formerly known as Swagger) documents.
   features:
     - description: >-
         The waterfall model does not work well for building APIs. Learning at the very end what your
```
Comment (paulatulis) [inline netlify/landings/prism.yaml]: I think an 'a' is missing from this sentence.
```suggestion
        The waterfall model does not work well for building APIs. Learning at the very end what your potential consumers think about your software can be a source of significant problems. 
```

File: netlify/landings/prism.yaml (hunk 1 above)
Comment (philsturgeon) [reply netlify/landings/prism.yaml]: I try and avoid sentences starting with And.

**Comment by philsturgeon** [comment]
@paulatulis great improvements, thanks for noticing all my mistakes! :D Feel free to jump in any time, no need to be invited.

**Comment by ksatx** [comment]
Call_me_note

**Comment by philsturgeon** [comment]
@ksatx ok! I think we're good to go here if you've got a spare thumb?

**Pull request merged**